pageserver/src/rocksdb_storage.rs

```diff
@@ -109,6 +109,40 @@ impl ObjectStore for RocksObjectStore {
 
         Ok(rels)
     }
+
+    fn objects<'a>(
+        &'a self,
+        timeline: ZTimelineId,
+        lsn: Lsn,
+    ) -> Result<Box<dyn Iterator<Item = Result<(BufferTag, Lsn, Vec<u8>)>> + 'a>> {
+        let start_key = StorageKey {
+            obj_key: ObjectKey {
+                timeline,
+                buf_tag: BufferTag {
+                    rel: RelTag {
+                        forknum: 0,
+                        spcnode: 0,
+                        dbnode: 0,
+                        relnode: 0,
+                    },
+                    blknum: 0,
+                },
```

Comment:
Should we give this creation a name, e.g. `const BufferTag::MIN` or `BufferTag::zeroed()`?